### ISSUE TITLE
Preact-router 2.4.2 fails to parse with webpack 2.3.2 and node 6.10.1

### ISSUE THREAD

**Issue by joemccall86** [developit/preact-router#167]
Our app stopped building roughly around when 2.4.2 was tagged. Our dependencies looked like this:

```
  "dependencies": {
    "preact": "^7.2.1",
    "preact-compat": "^3.14.3",
    "preact-router": "^2.4.1"
  }
```

Webpack failed with this message:

```
Module parse failed: /home/joe/code/github/my-project/node_modules/preact-router/src/index.js Unexpected token (253:28)
You may need an appropriate loader to handle this file type.
| }
| 
| const Route = ({ component, ...props }) => {
|       return h(component, props);
| };
```

Locking in 2.4.1 fixed the issue. Any ideas?

**Comment by developit** [comment]
Good timing - @lukeed just opened up a PR that fixes this: #168

**Comment by developit** [comment]
Should be fixed in 2.4.3 :)
